ddcz/models/used/users.py

```diff
@@ -60,23 +60,29 @@
 
 class UserProfile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE, blank=True, null=True)
-    jmeno_uzivatele = MisencodedCharField(max_length=20)
-    nick_uzivatele = MisencodedCharField(unique=True, max_length=25)
-    prijmeni_uzivatele = MisencodedCharField(max_length=20)
-    psw_uzivatele = MisencodedCharField(max_length=40)
-    email_uzivatele = MisencodedCharField(max_length=50)
-    pohlavi_uzivatele = MisencodedCharField(max_length=4, blank=True, null=True)
-    vek_uzivatele = models.IntegerField(default=0)
-    kraj_uzivatele = MisencodedCharField(max_length=20)
-    chat_barva = MisencodedCharField(max_length=6)
-    chat_pismo = models.IntegerField(default=12)
-    chat_reload = models.IntegerField(default=15)
-    chat_zprav = models.IntegerField(default=20)
-    chat_filtr = MisencodedCharField(max_length=255, blank=True, null=True)
-    chat_filtr_zobrazit = models.IntegerField(default=0)
-    pospristup = models.DateTimeField(auto_now_add=True)
-    level = MisencodedCharField(max_length=1)
-    icq_uzivatele = models.IntegerField(default=0)
+    nick = MisencodedCharField(unique=True, max_length=25, db_column="nick_uzivatele")
+    name_given = MisencodedCharField(max_length=20, db_column="jmeno_uzivatele")
+    name_family = MisencodedCharField(max_length=20, db_column="prijmeni_uzivatele")
+    password_v1 = MisencodedCharField(max_length=40, db_column="psw_uzivatele")
+    email = MisencodedCharField(max_length=50, db_column="email_uzivatele")
+    gender = MisencodedCharField(
+        max_length=4, blank=True, null=True, db_column="pohlavi_uzivatele"
+    )
+    age = models.IntegerField(default=0, db_column="vek_uzivatele")
+    shire = MisencodedCharField(max_length=20, db_column="kraj_uzivatele")
+    chat_color = MisencodedCharField(max_length=6, db_column="chat_barva")
+    chat_font = models.IntegerField(default=12, db_column="chat_pismo")
+    chat_reload = models.IntegerField(default=15, db_column="chat_reload")
+    chat_message_no = models.IntegerField(default=20, db_column="chat_zprav")
+    chat_filter = MisencodedCharField(
+        max_length=255, blank=True, null=True, db_column="chat_filtr"
+    )
+    chat_filter_display = models.IntegerField(
+        default=0, db_column="chat_filtr_zobrazit"
+    )
+    last_access = models.DateTimeField(auto_now_add=True, db_column="pospristup")
+    level = MisencodedCharField(max_length=1, db_column="level")
+    icq = models.IntegerField(default=0, db_column="icq_uzivatele")
```

Comment:
The most valuable information there ever been! :laughing:

Reply:
Right?